ts/WoltLabSuite/Core/Acp/Ui/User/Content/Remove/Handler.ts

```diff
@@ -24,13 +34,15 @@ interface AjaxResponse {
 class AcpUserContentRemoveHandler {
   private readonly dialogId: string;
   private readonly userId: number;
+  private readonly callbackSuccess: CallbackSuccess | null | undefined;
```

Comment:
```suggestion
  private readonly callbackSuccess: CallbackSuccess | null;
```

see below for the explanation.